osx/reset.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+#  This will remove the following.
+#    1. Unset environment variable: KALITE_PYTHON
+#    2. remove the .plist file, need admin
+#    3. delete the symlinked /usr/bin/kalite command, need admin
+
+KALITE_PLIST="/tmp/org.learningequality.kalite.plist"
+KALITE_PLIST_LIBRARY="/Library/LaunchAgents/org.learningequality.kalite.plist"
+KALITE_EXECUTABLE_PATH="$(which kalite)"
+
+
+# Delete the .plist file.
+if [ -f $KALITE_PLIST ]
+then
```

Comment:
Let's make our bash scripts consistent by using:

```bash
if [ boolean-statement ]; then
... blah
else
... blah
fi
```
Do the same with the others.